### PR TITLE
Rename the constant that refers to the `kubeadm-config` ConfigMap

### LINKED ISSUE #1096: Cleanup - Rename the constant that refers to the `kubeadm-config` ConfigMap
https://github.com/kubernetes/kubernetes/pull/67944 changed how kubeadm uses the `kubeadm-config`ConfigMap, that now stores only the `ClusterConfiguration` and the `ClusterStatus`.

We should now rename the constant that refers to the `kubeadm-config` from `InitConfigurationConfigMap` to something more generic like e.g. `KubeadmConfigConfigMap`

rif [code](
https://github.com/kubernetes/kubernetes/blob/e5f55dd9d0969af3a37e7880d9b88c9aef8c654d/cmd/kubeadm/app/constants/constants.go#L193)

### PR DESCRIPTION
**Pull request by RA489**
Cleanup - Rename the constant that refers to the kubeadm-config ConfigMap
fixes https://github.com/kubernetes/kubeadm/issues/1096

### CODE REVIEW

File: cmd/kubeadm/app/phases/uploadconfig/uploadconfig_test.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ func TestUploadConfiguration(t *testing.T) {
 				}
 			}
 			if tt.verifyResult {
-				masterCfg, err := client.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(kubeadmconstants.InitConfigurationConfigMap, metav1.GetOptions{})
+				masterCfg, err := client.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(kubeadmconstants.,KubeadmConfigConfigMap metav1.GetOptions{})
```
Comment (rosti) [inline cmd/kubeadm/app/phases/uploadconfig/uploadconfig_test.go]: Misplaced comma here is going to break this test, please fix it.

**Review by rosti — COMMENTED** [review]
LGTM from me (apart from the test breakage due to misplaced comma).
I was wondering if we should name it `KubeadmConfigMap`, because `KubeadmConfigConfigMap` sounds way too `Config` to me :)

/ok-to-test
/release-note-none
/assign @fabriziopandini

**Comment by rosti** [comment]
Looks great now!

/assign @timothysc

**Comment by fabriziopandini** [comment]
thanks for this PR!
/approve
/lgtm

**Pull request merged**